### ISSUE TITLE
SVG Render Glitch in Android

### ISSUE THREAD

**Issue by TomDino** [airbnb/lottie-web#1261]
My SVG animation works perfectly in all browsers except on mobile in Android (chrome, latest version). Some of the SVG paths or shapes don't render properly and only show a 1 pixel line where they should be.

The only way to stop this glitch is to remove the viewport meta tag, but I need this for the rest of the site to display properly so can't remove it:

`<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />`

Another solution I tried is to change the renderer from SVG to Canvas, this shows the missing shapes however some other shape glitches occur instead (black boxes around the shape).

[Screenshot showing it works on desktop](https://imgur.com/a/Du35nBe)
[Screenshot showing the SVG glitch on Android mobile](https://imgur.com/a/IUCG2bh)

Codepen here: https://codepen.io/anon/pen/RYmavX

**Comment by natedigrate** [comment]
Yes, this is happening to me to (especially the 1 pixel line part, which may be due to masking). Any help would be appreciated.

**Comment by bodymovin** [comment]
@TomDino Hi, can you share the .aep?

**Comment by jpetitte** [comment]
@TomDino did you ever arrive at a workable solution?

@bodymovin , I'm seeing the same thing. I can share our .aep if you would like.

**Comment by TomDino** [comment]
@jpetitte This turned out to be an issue with using masks in the graphic design software the illustrator was using, he just informed me that removing the mask then re-exporting it as JSON is what fixed this for us. Hope it helps!

**Comment by jpetitte** [comment]
@TomDino awesome, that's a place to start! Do you know if it was a specific type of mask? The more constraints I can give my illustrator, the more it will help him.

**Comment by TomDino** [comment]
@jpetitte It was in After Effects, it was either a mask or pre-comp that wasn't working, so they had to redo it as a regular vector. I'm not sure what the means myself, I'm just passing on what the illustrator told me as he passed my desk earlier, hope that helps!

**Comment by bodymovin** [comment]
@jpetitte it's probably related to alpha and inverted alpha masks, but I have to look into it.

**Comment by jpetitte** [comment]
@TomDino @bodymovin gotcha, I'll talk with my illustrator and report back what we discover.

**Comment by develth** [comment]
We have the same issue. We don´t use masks but a pre-comp. 
The crazy part is. When i open my devtool and inspect my mobile, the issue is gone as soon as i click ANYTHING in dev tools

**Comment by develth** [comment]
Tested it again without pre-comp - same issue
